pootle/core/mixins/treeitem.py

```diff
@@ -642,8 +662,10 @@ def create_job(self, status=None, depends_on=None):
         Creates Job object with given job ID
         """
         args = (self.instance,)
-        return Job.create(self.func, args=args, id=self.id, connection=self.connection,
-                          depends_on=depends_on, status=status)
+        return Job.create(
+            self.func, args=args,
+            id=self.id, connection=self.connection,
+            depends_on=depends_on, status=status)
```

Comment:
I remember this code as something that I had to read several times to understand how the stats caching worked, so I would prefer to keep this even more readable than it was. My suggestion is to put an argument per line if possible.